rca/programmes/models.py

```diff
@@ -417,6 +423,11 @@ class ProgrammePage(ContactFieldsMixin, BasePage):
         ],
         blank=True,
     )
+    qs_code = models.PositiveIntegerField(
+        help_text="This code needs to match the name of the codeExternal value in QS, E.G 105",
+        blank=True,
+        null=True,
```

Comment:
Null not needed for CharField

Reply:
This one is an integer field, and I need it to be null to control form options like this https://github.com/torchbox/rca-wagtail-2019/blob/feature/624-qs-integration/rca/enquire_to_study/forms.py#L36

```
programmes = forms.ModelMultipleChoiceField(
        queryset=ProgrammePage.objects.filter(qs_code__isnull=False, live=True),
        widget=forms.CheckboxSelectMultiple,
    )
```